app.json

```
@@ -37,6 +37,14 @@
     "SLACK_REQUEST_LOG_ENABLED": {
       "description": "Slack App Request Log Feature (0: disabled, 1: enabled)",
       "value": "0"
+    },
+    "SLACK_AUTO_TRANSLATE": {
+      "description": "Automatically translate between two languages (0: disabled, 1: enabled)",
+      "value": "0"
+    },
+    "SLACK_AUTO_TRANSLATE_LANG": {
```

This should be plural as there are two elements here
```suggestion
    "SLACK_AUTO_TRANSLATION_LANGUAGES": {
```